postgres/metadata.csv

```diff
@@ -78,5 +78,9 @@ postgresql.queries.local_blks_dirtied,count,,block,,"Total number of local block
 postgresql.queries.local_blks_written,count,,block,,"Total number of local blocks written per normalized query, database, and user. (DBM only)",0,postgres,postgres queries local blocks written
 postgresql.queries.temp_blks_read,count,,block,,"Total number of temp blocks read per normalized query, database, and user. (DBM only)",0,postgres,postgres queries temp blocks read
 postgresql.queries.temp_blks_written,count,,block,,"Total number of temp blocks written per normalized query, database, and user. (DBM only)",0,postgres,postgres queries temp blocks written
+postgresql.queries.duration.max,gauge,,,,The age of the longest running query (DBM only).,0,postgres,postgres queries duration max
```

Comment:
We should note which dimensions it's broken down by. Like for queries how we say "total number of temp blocks written **per normalized query, database, and user**"